tests/unit/dojson/test_dojson_utils.py

```diff
@@ -194,3 +195,47 @@ def test_classify_field_not_a_string():
     result = [classify_field(field) for field in raw_field_codes]
 
     assert expected == result
+
+
+def test_get_int_value_with_valid_input():
+    """Test int getting with valid string input."""
+    references = {'0': '1343079'}
+    expected = 1343079
+    result = get_int_value(references, '0')
+
+    assert expected == result
+
+def test_get_int_value_with_empty_string():
+    """Test int getting with empty string."""
+    references = {'0': ''}
+    expected = None
+    result = get_int_value(references, '0')
+
+    assert expected == result
```

Comment:
Finally, the data we are testing is small enough that it can fit on the same line, so let's do that:
```python
def test_get_int_value_returns_none_on_empty_string():
    assert get_int_value({'0': ''}, '0') is None
```